sdks/python/apache_beam/dataframe/frame_base.py

```diff
@@ -484,18 +484,43 @@ def args_to_kwargs(base_type):
   determine the name to use for arguments that are converted to keyword
   arguments.
 
-  For internal use only. No backwards compatibility guarantees."""
+  removed_method used in cases where a method has been removed in a later
+  version of Pandas. removed_args used in cases where a method has had
+  arguments removed in a later version of Pandas.
+
+  For internal use only. No backwards compatibility guarantees.
+  """
   def wrap(func):
-    arg_names = getfullargspec(unwrap(getattr(base_type, func.__name__))).args
+    if removed_method:
+      # Do no processing, let Beam function itself raise the error if called.
```

Comment:
will the error self-explanatory to the user?

Reply:
It will be a not implemented error saying that it isn't supported, even if I don't special case it for version deprecation that is still enough.